docs/blog/_posts/2017-05-31-first-dotty-milestone-release.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.1.2-RC1, a major step towards Scala 3
+author: Dmytro Petrashko
+authorImg: /images/petrashko.jpg
+---
+
+Today, we excited to release Dotty version 0.1.2-RC1.  This release
+serves as a technology preview that demonstrates new language features
+and the compiler supporting them.
+
+
+<!--more-->
+
+## Why is this important?
+
+_Dotty_ is the project name for a language and compiler that
+is slated to become Scala 3.0. This is an ongoing development,
+developed transparently as open source software. The Dotty project
+started more than 4 years ago. It reached a major milestone in 2015 by
+achieving
+[bootstrap](http://dotty.epfl.ch/blog/2015/10/23/dotty-compiler-bootstraps.html),
+that is, showing that the new compiler could compile itself.  Today we
+have reached another milestone with this first release of the
+codebase. Developments will not stop here, but they will in the future
+all be part of regular time-based releases.
+
+## How can you try it out?
+
+You have several alternatives: use the `sbt-dotty` plugin, get a standalone
+installation, or try it online on [Scastie](https://scastie.scala-lang.org/).
+
+### sbt
+Using sbt 0.13.13 or newer, do:
+
+```
+sbt new lampepfl/dotty.g8
+```
+
+This will setup a new sbt project with Dotty as compiler. For more details on
+using Dotty with sbt, see the
+[example project](https://github.com/lampepfl/dotty-example-project).
+
+### Standalone installation
+
+Releases are available for download on the _Releases_
+section of the Dotty repository: https://github.com/lampepfl/dotty/releases.
+
+We also provide a [homebrew](https://brew.sh/) package that can be installed by running
+
+```
+brew install lampepfl/brew/dotty
+```
+
+### Scastie
+
+[Scastie](https://scastie.scala-lang.org/), the online Scala playground,
+supports Dotty. To try it out:
+1. Click on `Build Settings` in the menu on the left
+2. Under `Target`, click on `Dotty`
+3. That's it!
+
+
+# What’s in the 0.1.2-RC1 technology preview ?
```

Comment:
Technology preview (TP) is a phase that comes a bit earlier than release candidate (RC). I know we are using the term here loosely, however.